apps/docs/src/app/platform/component-docs/platform-table/platform-table-examples/platform-table-navigatable-row-example.component.ts

```diff
@@ -0,0 +1,176 @@
+import { Component, ViewChild, AfterViewInit, ChangeDetectionStrategy } from '@angular/core';
+import { Observable, of } from 'rxjs';
+
+import { FdDate } from '@fundamental-ngx/core';
+import {
+    TableDataSource,
+    TableDataProvider,
+    TableState,
+    TableRowSelectionChangeEvent,
+    TableComponent
+} from '@fundamental-ngx/platform';
+
+@Component({
+    selector: 'fdp-platform-table-navigatable-row-example',
+    templateUrl: './platform-table-navigatable-row-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformTableNavigatableRowExampleComponent implements AfterViewInit {
+
+    source: TableDataSource<ExampleItem>;
+
+    @ViewChild(TableComponent)
+    table: TableComponent;
+
+    constructor() {
+        this.source = new TableDataSource(new TableDataProviderExample());
+    }
+
+    ngAfterViewInit(): void {
+        this.setRowUnnavigatable(0);
+        this.setRowUnnavigatable(1);
+        this.setRowUnnavigatable(2);
+    }
+
+    onRowSelectionChange(event: TableRowSelectionChangeEvent<ExampleItem>): void {
+        console.log(event);
```

Comment:
Does it needed here?

Reply:
I left this snipped in there because I saw it in the other examples. Just trying to following existing pattern.

Reply:
This snippet has been taken out as well.